From 123c0e91a7c6a7ef9604034728a295321a340e72 Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Wed, 7 Jun 2017 10:03:02 +0100
Subject: [PATCH 063/102] Revert "ASoC: Intel: Boards: Added option to enable
 TDF8532 DAI link"

This reverts commit cab970a7cb8be9c8289a751910c35cb8063f9b2d.

Conflicts:
	sound/soc/intel/Kconfig
---
 sound/soc/intel/boards/Kconfig     | 10 ----------
 sound/soc/intel/boards/bxt_gpmrb.c |  5 -----
 2 files changed, 15 deletions(-)

diff --git a/sound/soc/intel/boards/Kconfig b/sound/soc/intel/boards/Kconfig
index 8428d6e..987c808 100644
--- a/sound/soc/intel/boards/Kconfig
+++ b/sound/soc/intel/boards/Kconfig
@@ -417,16 +417,6 @@ config SND_SOC_INTEL_BXT_GPMRB_MACH
 	   Say Y if you have such a device
 	   If unsure select "N".
 
-config SND_SOC_INTEL_BXT_GPMRB_MACH_TDF8532_CODEC
-	bool "Enable TDF8532 codec"
-	depends on SND_SOC_INTEL_BXT_GPMRB_MACH
-	select SND_SOC_TDF8532
-	default y
-	help
-	   This adds support for TDF8532 codec to BE DAI link.
-	   Say "Y" if you are using GP MRB board.
-	   Otherwise choose "N".
-
 config SND_SOC_INTEL_APLI_LHCRB_DUMMY_MACH
 	tristate "ASoC Audio driver for APLI LHCRB Dummy Codec"
 	depends on X86_INTEL_LPSS && I2C
diff --git a/sound/soc/intel/boards/bxt_gpmrb.c b/sound/soc/intel/boards/bxt_gpmrb.c
index 8e816d5..968013a 100644
--- a/sound/soc/intel/boards/bxt_gpmrb.c
+++ b/sound/soc/intel/boards/bxt_gpmrb.c
@@ -505,13 +505,8 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.name = "SSP4-Codec",
 		.be_id = 0,
 		.cpu_dai_name = "SSP4 Pin",
-#ifdef CONFIG_SND_SOC_INTEL_BXT_GPMRB_MACH_TDF8532_CODEC
 		.codec_name = "i2c-INT34C3:00",
 		.codec_dai_name = "tdf8532-hifi",
-#else
-		.codec_name = "snd-soc-dummy",
-		.codec_dai_name = "snd-soc-dummy-dai",
-#endif
 		.platform_name = "0000:00:0e.0",
 		.ignore_suspend = 1,
 		.dpcm_playback = 1,
-- 
1.9.1

